src/index.html

```diff
@@ -1,17 +1,158 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" autocomplete="off"
+      onsubmit="onSubmit()" method="POST">
+      <fieldset class="field-set">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="forSurname"> Surname: </label>
+          <input type="text" name="Surname"
+           id="forSurname"
+           required>
```

Comment:
```suggestion
          <input
            type="text"
            name="Surname"
            id="surname"
            required
          >
```
Кожен атрибут на новому рядку. Закриваюча дужка на тому ж рівні відступів, що і відкриваюча

Атрибут id не потрібно починати зі слова for